lib/datasource/galaxy-collection/index.spec.ts

```diff
@@ -25,21 +25,22 @@ describe(getName(), () => {
     it('returns null for 404 result', async () => {
       httpMock.scope(baseUrl).get('/api/v2/collections/foo/bar/').reply(404);
       expect(
-        await getPkgReleases({ datasource, depName: 'foo.bar' })
+        await getPkgReleases({
+          datasource: GalaxyCollectionDatasource.id,
```

Comment:
can we add a modue scope `const datasource = GalaxyCollectionDatasource.id` so we have less changes ?